public/styles.css

```diff
@@ -0,0 +1,115 @@
+@font-face {
+    font-family: 'Amazon Ember';
+    src: url(https://m.media-amazon.com/images/G/01/AUIClients/AmazonUIFont-amazonember_rg-cc7ebaa05a2cd3b02c0929ac0475a44ab30b7efa._V2_.woff2);format("woff2"),url(https://m.media-amazon.com/images/G/01/AUIClients/AmazonUIFont-amazonember_rg-8a9db402d8966ae93717c348b9ab0bd08703a7a7._V2_.woff) format("woff")}
+    
+@font-face {
+    font-family: 'Amazon Ember';
+    font-style: italic;
+    src: url(https://m.media-amazon.com/images/G/01/AUIClients/AmazonUIFont-amazonember_rgit-9cc1bb64eb270135f1adf3a4881c2ee5e7c37be5._V2_.woff2) format("woff2"),url(https://m.media-amazon.com/images/G/01/AUIClients/AmazonUIFont-amazonember_rgit-a4dc98d644ff2aedd41da3da462f09ffce86eafb._V2_.woff) format("woff")}
+    
+@font-face {
+    font-family: 'Amazon Ember';
+    font-weight: 700;
+    src: url(https://m.media-amazon.com/images/G/01/AUIClients/AmazonUIFont-amazonember_bd-46b91bda68161c14e554a779643ef4957431987b._V2_.woff2) format("woff2"),url(https://m.media-amazon.com/images/G/01/AUIClients/AmazonUIFont-amazonember_bd-b605252f87b8b3df5ae206596dac0938fc5888bc._V2_.woff) format("woff")}
+    
+@font-face {font-family: 'Amazon Ember';
+    font-style: italic;
+    font-weight: 700;
+    src: url(https://m.media-amazon.com/images/G/01/AUIClients/AmazonUIFont-amazonember_bdit-80ff7aba37dd1ff5a6b90233a19e3a780a96dc2f._V2_.woff2) format("woff2"),url(https://m.media-amazon.com/images/G/01/AUIClients/AmazonUIFont-amazonember_bdit-57598ce426a612be5a1d15eee08252668fca5e7a._V2_.woff) format("woff")}
+    
+.a-ember body {
+    font-family:"Amazon Ember", Arial, sans-serif}
+    
+.a-ember
+.a-text-quote {
+    font-family: "Amazon Ember", Arial, sans-serif}
+
+
+#des_itemDescriptionModule {
+    display: flex;
+    width: 31%;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 5px;
+}
+
+#des_title {
+    font-family: "Amazon Ember";
+    font-size: 21px!important;
+    line-height: 1.3!important;
+    padding-bottom: 10px;
+
+}
+
+#des_reviewsRatingsQuestions{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    border-bottom: 1px solid rgb(202, 197, 197);
+    margin-bottom: 15px;
+}
+
+#des_rating {
+    font-family: "Amazon Ember";
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;  
+}
+
+#des_stars {
+    font-family: "Amazon Ember";
+    padding-right: 10px;
+}
+
+#des_numOfRatings {
+    font-family: "Amazon Ember";
+    color: royalblue;
+    font-size: 15px;
+
+}
+
+#des_numOfQuestions {
+    font-family: "Amazon Ember";
+    color: royalblue;
+    font-size: 15px;
+}
+
+#des_listPrice {
+    font-family: "Amazon Ember";
```

Comment:
You have font family declared in a lot of places, you only need to declare it once in the global wrapper. As per style guide, please refactor